blocks/form/submit.js

```diff
@@ -75,20 +75,16 @@ function constructPayload(form) {
 
 async function prepareRequest(form) {
   const { payload } = constructPayload(form);
-  const {
-    branch, site, org, tier,
-  } = getRouting();
   const headers = {
     'Content-Type': 'application/json',
-    'x-adobe-routing': `tier=${tier},bucket=${branch}--${site}--${org}`,
+    'x-adobe-form-hostname': window?.location?.hostname
   };
   const body = { data: payload };
   let url;
   let baseUrl = getSubmitBaseUrl();
-  if (!baseUrl && org && site) {
+  if (!baseUrl) {
     baseUrl = 'https://forms.adobe.com/adobe/forms/af/submit/';
-    headers['x-adobe-routing'] = `tier=${tier},bucket=${branch}--${site}--${org}`;
-    url = baseUrl + btoa(form.dataset.action);
+    url = baseUrl + btoa(form.dataset.action + '.json');
```

Comment:
Can you please also update submitFailure to use x-error header to show error if available

Reply:
@jalagari , i will raise a separate PR for this, will need to verify on ui add tests as well.